src/docfx/config/ops/OpsConfigAdapter.cs

```diff
@@ -76,8 +76,8 @@ private async Task<string> GetBuildConfig(Uri url)
     {
         var queries = HttpUtility.ParseQueryString(url.Query);
         var name = queries["name"] ?? "";
-        var repository = queries["repository_url"] ?? "";
-        var branch = queries["branch"] ?? "";
+        var repository = queries["publish_repository_url"] ?? queries["repository_url"] ?? "";
```

Comment:
>queries["publish_repository_url"] [](http://example.com/codeflow?start=25&length=33)

When there is no  `publish_repository_url`, there will be a qeury `publish_repository_url=&xxx`, they you will get a empty string instead of fallbacking to `repository_url`